corehq/apps/es/forms.py

```diff
@@ -41,6 +43,10 @@ def domain_aggregation(self):
         return self.terms_aggregation('domain', 'domain')
 
 
+def domains(domains):
+    return filters.term('domain', domains)
```

Comment:
This filter already exists it's just called `domain`: https://github.com/dimagi/commcare-hq/blob/d9f28ad6f0c337221bde07f7809c1517ce8b1098/corehq/apps/es/es_query.py#L442-L442